File: codes/cpp/Code_5_10_1.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
	long long a, b, c;
	cin >> a >> b >> c;
	if (sqrt(a) + sqrt(b) < sqrt(c)) cout << "Yes" << endl;
	else cout << "No" << endl;
	return 0;
}

File: codes/java/Code_5_10_1.java
import java.util.*;

class Code_5_10_1 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		long a = sc.nextLong();
		long b = sc.nextLong();
		long c = sc.nextLong();
		if (Math.sqrt(a) + Math.sqrt(b) < Math.sqrt(c)) {
			System.out.println("Yes");
		}
		else {
			System.out.println("No");
		}
	}
}
